services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/InterChange.java

```diff
@@ -0,0 +1,86 @@
+package com.mapbox.api.directions.v5.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.mapbox.api.directions.v5.DirectionsAdapterFactory;
+
+/**
```

Comment:
```suggestion
/*
```
Same everywhere else.

Reply:
Is this supposed to be hidden? Otherwise this should be a Javadoc comment for class

Reply:
Done. Made the passive maneuvers hidden.